Automattic/jetpack
# Jetpack Forms: Change the message sent Tracks event to a bump stat

The message sent event is a Tracks event but it really should be a bump stat.

This PR converts the contact_form_block_message_sent Tracks event to a bump stat called "jetpack_forms_message_sent". The extra value of the stat is the publish week of the post containing the form.

### Other information:


## Jetpack product discussion
panfyZ-1VS-p2#comment-4776

## Does this pull request change what data or activity we track or use?
Yes, it reduces the amount of data we track.

## Testing instructions:
Apply patch
It might be easier to test if you add an error_log() to jetpack_tracks_record_grunion_pre_message_sent() so you see the code is reached.
Create a form on a post.
Fill in the form and submit it.
Check your php error_log.
Automatticians can check the bump stat for jetpack_forms_message_sent.
